Makefile

```diff
@@ -143,7 +144,7 @@ image-build: image-build-ansible image-build-helm image-build-scorecard-proxy ##
 image-build-ansible: build/operator-sdk-dev-x86_64-linux-gnu
```

Comment:
Any reason why we can't do the same for helm images as ansible?

Reply:
Ansible is already using `GOOS=linux GOARCH=amd64`, so it is actually more strict. The reason we don't use `amd64` for helm is because we support a `ppc64le` base image.

Reply:
It is working for ansible. I just re-tested it as well.